uberenv.py

```diff
@@ -226,9 +226,13 @@ def parse_args():
     return opts, extras
 
 
+def pretty_print_dictionary(dictionary):
+    for key, value in dictionary.items():
+        print("  {0}: {1}".format(key, value))
+
 def uberenv_script_dir():
     # returns the directory of the uberenv.py script
-    return os.path.dirname(os.path.realpath(__file__))
+    return os.path.dirname(os.path.abspath(__file__))
```

Comment:
@kennyweiss this was causing problems when finding the config file in the parent directory when it undid the symlink.